pkg/trace/api/dogstatsd.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 // dogstatsdProxyHandler returns a new HTTP handler which will proxy requests to
-// the DogStatsD endpoint in the Core Agent over UDP or UDS (defaulting to UDS
-// if StatsdSocket is set in the *AgentConfig).
+// the DogStatsD endpoint in the Core Agent over UDP.
+// UDS is not supported, see https://github.com/DataDog/datadog-agent/pull/13628
```

Comment:
>  UDS is not supported, see https://github.com/DataDog/datadog-agent/pull/13628

This could imply to users that the proxy shouldn't be connected to over UDS, which is not true. It's more an implementation detail of how the proxy works. Perhaps we should just remove the 
> or UDS (defaulting to UDS if StatsdSocket is set in the *AgentConfig).

since the distinction isn't important to users of the proxy.

Reply:
Actually, users might care that stats are delivered to the core agent over UDP instead of UDS, since UDP is not guaranteed to deliver.

Maybe we should say something along the lines of
> // dogstatsdProxyHandler returns a new HTTP handler which will proxy requests to
// the DogStatsD endpoint in the Core Agent over UDP. Communication between the 
// proxy and the agent does not support UDS, and so does not guarantee delivery of
// all statsd payloads.

Reply:
Agreed - thanks for the suggestion! (applied it)